tests/python/pants_test/pantsd/test_pantsd_integration.py

```diff
@@ -304,13 +307,22 @@ def full_pantsd_log():
       )
 
       checker.assert_running()
+
       touch(test_file)
       # Permit ample time for the async file event propagate in CI.
       time.sleep(10)
+      checker.assert_running()
+
+      # Update test_file2
+      safe_file_dump(test_file2, "import that\n", mode='w')
+      time.sleep(10)
       checker.assert_stopped()
 
       self.assertIn('saw file events covered by invalidation globs', full_pantsd_log())
 
+      # Remove test_file2
+      safe_delete(test_file2)
```

Comment:
A useful facility to create temporary files is the [`temporary_file()`](https://github.com/pantsbuild/pants/blob/8b338327d65381148cd2a99c1292cde19580171d/src/python/pants/util/contextutil.py#L210) context manager.

Reply:
+1... this should use some sort of context manager for the created file. Would do something like:
```
with temporary_file(suffix='.py') as temp_f:
  temp_f.write("import that\n")
  temp_f.close()

  time.sleep(10)
  checker.assert_stopped()
  ...
```

Reply:
Done